landoapi/app.py

```diff
@@ -24,9 +29,17 @@ def create_app(version_path):
 
     # Get the Flask app being wrapped by the Connexion app.
     flask_app = app.app
+
     flask_app.config['VERSION_PATH'] = version_path
     log_config_change('VERSION_PATH', version_path)
 
+    version_info = json.load(open(version_path))
+    logger.info(version_info, 'app.version')
+
+    this_app_version = version_info['commit']
+    environment = version_info['build']
```

Comment:
The version.json 'build' doesn't refer to the environment, the refers to the specific cirlce ci build that created this image (see https://circleci.com/docs/1.0/environment-variables/). With that in mind it cannot be used as the environment. 

We could create another environment variable ('ENVIRONMENT' or something) and have @ckolos  set it properly in dev, staging, and prod. Or we can switch based on the HOST_URL that is already available.